xtime/xtime.go

```diff
@@ -84,6 +84,24 @@ type TimeTree struct {
 	root    timeNode
 }
 
+func (tt *TimeTree) ParseInLocation(t string, loc *time.Location) (time.Time, error) {
```

Comment:
~~~What is the behavior is `loc` is nil?~~~

Edit: Looks like it'll probably just be the default behavior of `time.ParseInLocation` which makes sense.